src/Stryker.Core/Stryker.Core.UnitTest/MutantFilters/DiffMutantFilterTests.cs

```diff
@@ -218,7 +220,7 @@ public void ShouldNotFilterMutantsWhereCoveringTestsContainsChangedTestFile()
         {
             // Arrange
             string testProjectPath = "C:/MyTests";
-            var options = new StrykerOptions();
+            var options = new StrykerOptions(diff: false);
```

Comment:
Setting diff to false here makes no sense as this test is about testing the diff feature

Reply:
the other tests in the class have the same option

Reply:
https://github.com/stryker-mutator/stryker-net/blob/bbbc78eeb98d2f19a763afbb7d03054dffa8b6cc/src/Stryker.Core/Stryker.Core.UnitTest/MutantFilters/DiffMutantFilterTests.cs#L96

Reply:
Mhm it seems some tests have it set to false and some don't. Interesting. In any case the original did not have diff set to false, so why are you changing it?